recipes/threadpoolctl/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set version = "2.0.0" %}
+
+package:
+  name: threadpoolctl
+  version: {{ version }}
+
+source:
+  url: https://github.com/joblib/threadpoolctl/archive/{{ version }}.tar.gz
+  sha256: 49b987e9e86168eec68d0c706091bd6a8590268abc249b32bcd2a3909d331a04
+
+build:
+  number: 0
+  skip: true  # [py<35]
+  script:
+    # Tests needs compiled extensions and are not packaged with threadpoolctl.
+    # We need to run them at build time.
+    - bash ./continuous_integration/build_test_ext.sh
+    - pytest -vl
+    - {{ PYTHON }} -m flit install
+
+requirements:
+  build:
+    - cython >=0.28.5
+    - scipy >=0.17
+    - {{ compiler('c') }}
+    - llvm-openmp  # [osx]
+    - pytest >=3.3.0
+  host:
+    - python >=3.5
+    - flit
+  run:
+    - python >=3.5
+
+about:
+  home: https://github.com/joblib/threadpoolctl
+  license: BSD
+  license_family: BSD
+  license_file: LICENSE
+  summary: 'Python helpers to control the threadpools of native libraries'
```

Comment:
We need a quick description of the package. I put a shorten version of the description (below) but it's not very catchy. Do you have any suggestion ?

Reply:
Looks good to me.

Reply:
Looks fine!